### ISSUE TITLE
NuGet config packageSource should not be local

### ISSUE THREAD

**Issue by Corniel** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#640]
Local paths should be avoided:
``` xml
<configuration>
  <packageSources>
    <clear />
    <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
    <add key="LOCAL" value="C:\code\_packages" />
  </packageSources>
</configuration>

```

**Comment by Corniel** [comment]
@Sjaaky As long as we can not think of any (likely) scenario where this would be okay for production code/apps, this rule should be totally valid.
